src/main/java/com/mathworks/ci/helper/MatlabBuild.java

```diff
@@ -70,7 +73,7 @@ default String getPlatformSpecificRunner(File tempDirectory) throws IOException
      */
     default void copyFileInWorkspace(String sourceFile, File targetWorkspace) throws IOException {
         final ClassLoader classLoader = getClass().getClassLoader();
-        final File destination = new File(targetWorkspace, sourceFile);
+        final File destination = new File(targetWorkspace, new File(sourceFile).getName());
```

Comment:
Why do we have to do `new File(sourceFile).getName()` now? Seems like we're converting a string to a File, back to a string, back to a File... whew that's a mouthful

Reply:
Yeah looks good! I think we could go ahead and change it now.